ouroboros-network/src/Ouroboros/Network/PeerSelection/State/LocalRootPeers.hs

```diff
@@ -46,7 +49,7 @@ data LocalRootPeers peeraddr =
        -- We use two partial & overlapping representations:
 
        -- The collection of all the peers, with the associated PeerAdvertise
-       (Map peeraddr PeerAdvertise)
+       (Map peeraddr (PeerAdvertise, PeerTrustable))
 
        -- The groups, but without the associated PeerAdvertise
```

Comment:
`PeerTrustable` will also be needed to be accessed per peer, since we want to be able to simply distinguish which peers are trustable or not, so although they are a property of the group (like `advertise` is), I think keeping things this way is the way to go